generated/python/grpc-google-service-logging-types-v1/setup.py

```diff
@@ -0,0 +1,43 @@
+"""A setup module for the GRPC Stackdriver Logging External Types service.
+
+See:
+https://packaging.python.org/en/latest/distributing.html
+https://github.com/pypa/sampleproject
+"""
+
+import setuptools
+
+from setuptools import setup, find_packages
+
+install_requires = [
+  'oauth2client>=2.0.0, <4.0dev',
+  'grpcio>=1.0.2, <2.0dev',
+  'googleapis-common-protos[grpc]>=1.5.0, <2.0dev'
+]
+
+setuptools.setup(
+  name='grpc-google-cloud-service-logging-types-v1',
+  version='0.14.0',
+  author='Google Inc',
+  author_email='googleapis-packages@google.com',
+  classifiers=[
+    'Intended Audience :: Developers',
+    'Development Status :: 3 - Alpha',
+    'Intended Audience :: Developers',
+    'License :: OSI Approved :: Apache Software License',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 2',
+    'Programming Language :: Python :: 2.7',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.4',
+    'Programming Language :: Python :: 3.5',
+    'Programming Language :: Python :: Implementation :: CPython',
+  ],
+  description='GRPC library for the Stackdriver Logging External Types service',
+  long_description=open('README.rst').read(),
+  install_requires=install_requires,
+  license='Apache-2.0',
+  packages=find_packages(),
+  namespace_packages=['google.cloud.grpc.appengine.legacy', 'google.cloud.grpc.appengine.logging', 'google.cloud.grpc.appengine', 'google.cloud.grpc', 'google.cloud', 'google'],
```

Comment:
We don't currently declare version-number packages as namespace packages; this logic originated in [Packman](https://github.com/googleapis/packman/blob/378f7c479df0e0f7d243d06d48c4a60db5c47621/lib/packager.js#L223), and I later ported the same behavior to toolkit. I can't currently think of any reason to keep this behavior, though -- I'll file an issue in toolkit and we can discuss there whether this should be changed.